Source/Rio.API/Models/ParcelLedgerCreateCSVUploadDto.cs

```diff
@@ -10,6 +10,7 @@ public class ParcelLedgerCreateCSVUploadDto
         public IFormFile UploadedFile { get; set; }
         [Required]
         public DateTime EffectiveDate { get; set; }
+        public String EffectiveDateString { get; set; }

```

Comment:
lower case string
also I need to understand why you have both a DateTime version and a string version